CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [0.20.0](https://github.com/openeuropa/oe_translation/tree/0.20.0) (2022-03-21)
+[Full Changelog](https://github.com/openeuropa/oe_translation/compare/0.19.0...0.20.0)
+
+**Merged pull requests:**
+
+- EWPP-2908: Update to PHP8.1 and D9.4. [\#193](https://github.com/openeuropa/oe_translation/pull/193) ([22Alexandra](https://github.com/22Alexandra))
+- Prepare release 0.19.0. [\#178](https://github.com/openeuropa/oe_translation/pull/178) ([imanoleguskiza](https://github.com/imanoleguskiza))
```

Comment:
Do we need this? Since it's written manually anyway.

Reply:
Up there it says merged pull requests, it's a merged pull request and before we have it so out of consistency I would keep it.